### ISSUE TITLE
Skewer on its Last Use (or 2nd to Last Use) Yields But One Rubber Dough

### ISSUE THREAD

**Issue by DougLefelhocz** [jasonrohrer/OneLifeData7#874]
I've had this a few times where I haven't gotten a full 4 rubber dough from a bucket.  I just got one rubber dough (and thus only one tire).  The skewer I used, I had used to farm before.  It had a black spot in the middle (not sure if it was the first or second indicator).  I was able to make but one tire and nothing else.

**Comment by DougLefelhocz** [comment]
I reported a [similar issue ](https://github.com/jasonrohrer/OneLife/issues/506)back in January.  I closed it, because I couldn't reproduce in the same location and believed it was because I was using two biomes.  I had the same issue happen a few months ago, but didn't report it, as I couldn't reproduce it.  Each time this has happened, _it's been the first time I've cooked rubber._  My pattern of development has been to farm using skewers until the skewers have a notch in the middle and then not use the skewer, or hope it doesn't break (it seems to consistently have one more use after having the notch in the middle).  Then one of those "last use" skewers becomes a skewer I use when cooking rubber.  That then becomes a weak skewer afterwards.  Thus, the issue seems to have to do with "last use" skewers, or in other words skewers which have a notch in the middle and would soon break if tilled once or a few times more.
